src/main/java/cinema/project/model/CinemaHall.java

```diff
@@ -0,0 +1,50 @@
+package cinema.project.model;
+
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.Table;
+
+@Entity
+@Table(name = "cinemaHall")
```

Comment:
let's use the same type of table namings. If you have movie_session table, then let's name this one `cinema_hall`